### PR TITLE
Fix issue due to Conda-forge

### PR DESCRIPTION
**Pull request by licode**
also corrected sha256 number, and remove unused tests.

### CODE REVIEW

**Comment by licode** [comment]
Now conda auto-build is back again after a quick fix.

File: scripts/nsls2-dev-build.sh [hunk 1]
```diff
@@ -32,8 +32,8 @@ show_channel_urls: true
 channels:
 - $UPLOAD_CHANNEL
 - lightsource2-tag
-- conda-forge
-- defaults" > $CONDARC_PATH
+- defaults
+- conda-forge" > $CONDARC_PATH
```
Comment (tacaswell) [inline scripts/nsls2-dev-build.sh]: This seems a bit dangerous as we might get versions of things from conda-forge that have matching deps on conda-forge, but we will use the defaults version of the dep which may be older.

Comment (licode) [inline scripts/nsls2-dev-build.sh]: That's also one of my concerns. But this is a temporary solution after discussing with conda-forge team. We will move back when conda team fixed the bugs.

Comment (tacaswell) [inline scripts/nsls2-dev-build.sh]: sounds good.  Trust them over me here.

Comment (ericdill) [inline scripts/nsls2-dev-build.sh]: What are these bugs?

**Comment by licode** [comment]
Thanks for checking on this. Here is the error info
```
latest: Pulling from nsls2/debian-with-miniconda
Digest: sha256:362eb464ac1e3f283378dbce0384d1e37328cfdf64f7cc4da9307f55a7d1d1d7
Status: Image is up to date for nsls2/debian-with-miniconda:latest
Running the docker container
CONDARC_PATH=/root/.condarc
Exporting CONDARC=/root/.condarc
showing conda info
Current conda install:

             platform : linux-64
        conda version : 4.0.5
  conda-build version : not installed
       python version : 3.5.1.final.0
     requests version : 2.9.1
     root environment : /conda  (writable)
  default environment : /conda
     envs directories : /conda/envs
        package cache : /conda/pkgs
         channel URLs : https://conda.anaconda.org/lightsource2-tag/linux-64/
                        https://conda.anaconda.org/lightsource2-tag/noarch/
                        https://conda.anaconda.org/conda-forge/linux-64/
                        https://conda.anaconda.org/conda-forge/noarch/
                        https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /root/.condarc
    is foreign system : False

contents of condarc at /root/.condarc
binstar_upload: false
always_yes: true
show_channel_urls: true
channels:
- lightsource2-tag
- conda-forge
- defaults
Fetching package metadata: ........
Solving package specifications: .........

Package plan for installation in environment /conda:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ncurses-5.9                |                9         954 KB  conda-forge
    patchelf-0.9               |                1         349 KB  conda-forge
    sqlite-3.13.0              |                0         4.9 MB  conda-forge
    xz-5.2.2                   |                0         853 KB  conda-forge
    python-3.5.2               |                3        31.8 MB  conda-forge
    conda-env-2.5.1            |           py35_0          27 KB  defaults
    markupsafe-0.23            |           py35_2          32 KB  defaults
    psutil-4.3.1               |           py35_0         228 KB  defaults
    pytz-2016.6.1              |           py35_0         162 KB  conda-forge
    ruamel_yaml-0.11.14        |           py35_0         377 KB  defaults
    six-1.10.0                 |           py35_0          18 KB  conda-forge
    clyent-1.2.2               |           py35_0          15 KB  defaults
    conda-4.1.12               |           py35_0         204 KB  defaults
    jinja2-2.8                 |           py35_1         278 KB  conda-forge
    python-dateutil-2.5.3      |           py35_0         237 KB  conda-forge
    anaconda-client-1.4.0      |           py35_0         101 KB  defaults
    conda-build-1.21.3         |           py35_0         238 KB  defaults
    conda-execute-0.6.0        |           py35_0          14 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        40.7 MB

The following NEW packages will be INSTALLED:

    anaconda-client: 1.4.0-py35_0    defaults
    clyent:          1.2.2-py35_0    defaults
    conda-build:     1.21.3-py35_0   defaults
    conda-execute:   0.6.0-py35_0    conda-forge
    jinja2:          2.8-py35_1      conda-forge
    markupsafe:      0.23-py35_2     defaults
    ncurses:         5.9-9           conda-forge
    patchelf:        0.9-1           conda-forge
    psutil:          4.3.1-py35_0    defaults
    python-dateutil: 2.5.3-py35_0    conda-forge
    pytz:            2016.6.1-py35_0 conda-forge
    ruamel_yaml:     0.11.14-py35_0  defaults
    six:             1.10.0-py35_0   conda-forge

The following packages will be UPDATED:

    conda:           4.0.5-py35_0 defaults    --> 4.1.12-py35_0   defaults
    conda-env:       2.4.5-py35_0 defaults    --> 2.5.1-py35_0    defaults
    python:          3.5.1-0      conda-forge --> 3.5.2-3         conda-forge
    sqlite:          3.9.2-0      conda-forge --> 3.13.0-0        conda-forge
    xz:              5.0.5-1      conda-forge --> 5.2.2-0         conda-forge

...

   Building: linux-64/metadataclient-0.1.1-py34_0.tar.bz2
Build cmd: conda build /repo/recipes-tag/metadataclient --python 3.4 --numpy 1.11
BUILD START: metadataclient-0.1.1-py34_0
    (actual version deferred until further download or env creation)
updating index in: /conda/tmp_envs/55fc582f7dd397d3a5f0/pkgs
updating index in: /conda/tmp_envs/55fc582f7dd397d3a5f0/conda-bld/linux-64
updating index in: /conda/tmp_envs/55fc582f7dd397d3a5f0/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2016.8.31  |                0         163 KB  conda-forge
    ncurses-5.9                |                9         954 KB  conda-forge
    sqlite-3.13.0              |                1         4.9 MB  conda-forge
    tk-8.5.19                  |                0         1.9 MB  conda-forge
    xz-5.2.2                   |                0         853 KB  conda-forge
    zlib-1.2.8                 |                3          87 KB  conda-forge
    openssl-1.0.2h             |                2         3.4 MB  conda-forge
    readline-6.2               |                0         713 KB  conda-forge
    python-3.4.5               |                2        26.9 MB  conda-forge
    certifi-2016.8.31          |           py34_0         220 KB  conda-forge
    setuptools-26.1.1          |           py34_0         354 KB  conda-forge
    wheel-0.29.0               |           py34_0          82 KB  conda-forge
    pip-8.1.2                  |           py34_0         1.6 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        42.0 MB

The following NEW packages will be INSTALLED:

    ca-certificates: 2016.8.31-0      conda-forge
    certifi:         2016.8.31-py34_0 conda-forge
    ncurses:         5.9-9            conda-forge
    openssl:         1.0.2h-2         conda-forge
    pip:             8.1.2-py34_0     conda-forge
    python:          3.4.5-2          conda-forge
    readline:        6.2-0            conda-forge
                                                                                                               824,0-1       19%

os.uname(): posix.uname_result(sysname='Linux', nodename='763b6f905bd2', release='3.19.0-59-generic', version='#65-Ubuntu SMP Mon Apr 18 16:12:07 UTC 2016', machine='x86_64')

sys.version: 3.5.2 | packaged by conda-forge | (default, Sep  8 2016, 14:23:11)
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)]

log file: /root/auto-build-logs/09.12-02.51
Building: linux-64/metadataclient-0.1.1-py34_0.tar.bz2
Build cmd: conda build /repo/recipes-tag/metadataclient --python 3.4 --numpy 1.11
BUILD START: metadataclient-0.1.1-py34_0
    (actual version deferred until further download or env creation)
updating index in: /conda/tmp_envs/55fc582f7dd397d3a5f0/pkgs
updating index in: /conda/tmp_envs/55fc582f7dd397d3a5f0/conda-bld/linux-64
updating index in: /conda/tmp_envs/55fc582f7dd397d3a5f0/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2016.8.31  |                0         163 KB  conda-forge
    ncurses-5.9                |                9         954 KB  conda-forge
    sqlite-3.13.0              |                1         4.9 MB  conda-forge
    tk-8.5.19                  |                0         1.9 MB  conda-forge
    xz-5.2.2                   |                0         853 KB  conda-forge
    zlib-1.2.8                 |                3          87 KB  conda-forge
    openssl-1.0.2h             |                2         3.4 MB  conda-forge
    readline-6.2               |                0         713 KB  conda-forge
    python-3.4.5               |                2        26.9 MB  conda-forge
    certifi-2016.8.31          |           py34_0         220 KB  conda-forge
    setuptools-26.1.1          |           py34_0         354 KB  conda-forge
    wheel-0.29.0               |           py34_0          82 KB  conda-forge
    pip-8.1.2                  |           py34_0         1.6 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        42.0 MB

The following NEW packages will be INSTALLED:

    ca-certificates: 2016.8.31-0      conda-forge
    certifi:         2016.8.31-py34_0 conda-forge
    ncurses:         5.9-9            conda-forge
    openssl:         1.0.2h-2         conda-forge
    pip:             8.1.2-py34_0     conda-forge
    python:          3.4.5-2          conda-forge
    readline:        6.2-0            conda-forge
    setuptools:      26.1.1-py34_0    conda-forge
    sqlite:          3.13.0-1         conda-forge
    tk:              8.5.19-0         conda-forge
    wheel:           0.29.0-py34_0    conda-forge
                                                                                                                    824,0-1       19%
    certifi:         2016.8.31-py34_0 conda-forge
    ncurses:         5.9-9            conda-forge
    openssl:         1.0.2h-2         conda-forge
    pip:             8.1.2-py34_0     conda-forge
    python:          3.4.5-2          conda-forge
    readline:        6.2-0            conda-forge
    setuptools:      26.1.1-py34_0    conda-forge
    sqlite:          3.13.0-1         conda-forge
    tk:              8.5.19-0         conda-forge
    wheel:           0.29.0-py34_0    conda-forge
    xz:              5.2.2-0          conda-forge
    zlib:            1.2.8-3          conda-forge



========== STDOUT ==========

None

========== STDERR ==========

('Using Anaconda API: https://api.anaconda.org\n'
 'WARNING:/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.5/site-packages/conda_build/build.py:Build '
 'prefix failed with prefix length 255\n'
 'WARNING:/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.5/site-packages/conda_build/build.py:Error '
 'was: \n'
 'WARNING:/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.5/site-packages/conda_build/build.py:ERROR: '
 'placeholder '
 "'/opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho' "
 'too short in: conda-forge::ncurses-5.9-9\n'
 '\n'
 'WARNING:/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.5/site-packages/conda_build/build.py:One '
 'or more of your package dependencies needs to be rebuilt with a longer '
 'prefix length.\n'
 'WARNING:/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.5/site-packages/conda_build/build.py:Falling '
 'back to legacy prefix length of 80 characters.\n'
 'WARNING:/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.5/site-packages/conda_build/build.py:Your '
 'package will not install into prefixes > 80 characters.\n'
 'ERROR: placeholder '
 "'/opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho' "
 'too short in: conda-forge::ncurses-5.9-9\n'
 '\n')
Sending the following message to slack
```

**Comment by licode** [comment]
It looks that the issue has been resolved by conda team. We can move conda-forge in front of defaults now.

**Comment by ericdill** [comment]
I *think* that problem was coming from the conda 4.0.x binaries. Updating conda to 4.1.x or 4.2.x might fix it.

**Comment by ericdill** [comment]
Also there is a tremendously long issue where it is being discussed how to handle the change in the prefix length: https://github.com/conda-forge/conda-forge.github.io/issues/171#issuecomment-248400560

**Comment by licode** [comment]
I was using conda 4.1.2, but still got the error. I think the issue came from pytz and sqlite, which need to be installed from default, instead of conda-forge, according to conda-forge team.

However, those two packages have been fixed for now at conda-forge. Thanks for the follow up!

**Pull request merged**